docs/docs/deploying-to-now.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Deploying to Vercel Now
+---
+
```

Comment:
Since there is a rebranding change can we add a note that states that here and also a link to the official release documentation?  Thanks.

Reply:
Hmm, how do you mean?

Reply:
Like a note that points to https://vercel.com/blog/zeit-is-now-vercel for further information.

Reply:
I went ahead to add that in

Reply:
Awesome! Thanks